kazam/frontend/indicator.py

```diff
@@ -150,7 +150,9 @@ def on_menuitem_quit_activate(self, menuitem):
         self.emit("indicator-quit-request")
 
 try:
-    from gi.repository import AppIndicator3
+    import gi
+    gi.require_version('AyatanaAppIndicator3', '0.1')
```

Comment:
This would cause exception: `ValueError: Namespace AyatanaAppIndicator3 not available` in some cases.

Can you try catch it and use `from gi.repository import AppIndicator3` as a fallback?

Reply:
Probably this is not need if you add `gir1.2-ayatanaappindicator3-0.1` in debian/control as Depends field (or Requires for rpm-spec).
If I understand the code correctly - the fallback is already defined at https://github.com/N0rbert/kazam/blob/master/kazam/frontend/indicator.py#L230 .